apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktRoute.kt

```diff
@@ -288,30 +293,48 @@ internal fun Route.aktivitetspliktRoutes(
         }
     }
 
-    route("/api/aktivitetsplikt/oppgave/{${OPPGAVEID_CALL_PARAMETER}}") {
-        get {
-            val oppgaveOgVurdering = inTransaction { aktivitetspliktOppgaveService.hentVurderingForOppgave(oppgaveId) }
-            call.respond(oppgaveOgVurdering)
-        }
-        post("brevdata") {
-            val brevdata = call.receive<AktivitetspliktInformasjonBrevdataRequest>()
-            val oppdaterBrevdata = inTransaction { aktivitetspliktOppgaveService.lagreBrevdata(oppgaveId, brevdata) }
-            call.respond(oppdaterBrevdata)
-        }
-        post("opprettbrev") {
-            val brevId =
-                inTransaction {
-                    aktivitetspliktOppgaveService.opprettBrevHvisKraveneErOppfyltOgDetIkkeFinnes(oppgaveId = oppgaveId, brukerTokenInfo)
-                }
-            call.respond(BrevIdDto(brevId))
-        }
-        post("ferdigstillbrev-og-oppgave") {
-            val oppgave = inTransaction { aktivitetspliktOppgaveService.ferdigstillBrevOgOppgave(oppgaveId, brukerTokenInfo) }
-            call.respond(oppgave)
+    route("/api/aktivitetsplikt/oppgave") {
+        post("/opprett") {
```

Comment:
@oyvindsh skal vi kalle denne fra de automatiske jobbene også?

Reply:
Hmm, kanskje? Jobben må i så fall håndtere å få forskjellige forespoerselexceptions tilbake forskjellig fra internfeil, slik at den gir seg hvis den ikke skal opprette og prøver på nytt (kræsjlooper) hvis det er noe feil med behandling. Kanskje det letteste er å ha to endepunkt og så kan sjekkene hentes ut i en felles metode, og så kan oppgavene for de automatiske bare fange exceptions i service der, og saksbehandler-kallet sender de ut.

Reply:
se gjerne på https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/7044
merger de samtidig tenker jeg med denne først
og deretter den ovennevnte så sperrer på brev